backend/src/main/java/org/sefglobal/core/fellowship/controller/CertificateController.java

```diff
@@ -0,0 +1,24 @@
+package org.sefglobal.core.fellowship.controller;
+
+import org.sefglobal.core.exception.ResourceNotFoundException;
+import org.sefglobal.core.fellowship.model.Certificate;
+import org.sefglobal.core.fellowship.service.CertificateService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/fellowship/certificates")
+public class CertificateController {
+
+    private final CertificateService certificateService;
+
+    public CertificateController(CertificateService certificateService) {
+        this.certificateService = certificateService;
+    }
+
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public Certificate getCertificateById(@PathVariable(value = "id") Integer id) throws ResourceNotFoundException{
```

Comment:
```suggestion
    public Certificate getCertificateById(@PathVariable Integer id) throws ResourceNotFoundException{
```

Reply:
On it!

Reply:
Resolved with the commit [ef83b38](https://github.com/sef-global/sef-core/pull/98/commits/ef83b38b1656806a7b1d9cb336d12646c3d95ec3)